server/doc/swdesign/README.md

```diff
@@ -309,7 +309,7 @@ Needs:
 
 Status: approved
 
-When an Agent connects to the Server, the Ankaios Server shall send a list of all scheduled Workloads to this Agent as `AddedWorkload` message.
+When an Agent connects to the Server, the Ankaios Server shall send a list of all scheduled Workloads to this Agent as `ServerHello` message.
```

Comment:
We need to change the version of the requirement too.

Reply:
Done.